README.md

```diff
@@ -89,17 +97,35 @@ You can activate the Conda environment with:
 conda activate rcrunch
 ```
 
-### 5. Execution of RCRUNCH
+### 5. Test 
+To ensure that the version is working properly you can test it by:
+1. Activate the Conda environment with:
+```bash
+conda activate rcrunch
+```
+2. Run:
+```bash
+bash test/test_singularity_execution/test_local.sh
+```
+> ✨ Note: This test has a running time of ~8 minutes when using 4 cores (Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz)
+
+or for **SLURM** workload manager,
+```bash
+bash test/test_singularity_execution/test_slurm.sh
+```
 
 
-### 5a. Fill in the necessary config file
+### 6. Execution of RCRUNCH
+
+
+### 6a. Fill in the necessary config file
 
 In order to run RCRUNCH, please fill in the organism related data and the experiment-dependent parameters for the different samples in the file `config.yaml`.
 
 > ✨ For your convenience an empty [config.yaml](config.yaml) file is available to fill in.
 
 
-### 5b. Dry run and DAG generation (optional)
+### 6b. Dry run and DAG generation (optional)
```

Comment:
Explain what "dry run" is, e.g. "To check whether the pipeline can run and all of the dependencies are satisfied, without actually executing the commands, you can do a 'dry run' with the following command:

This command will generate a graph (directed acyclic graph, DAG) that shows you how the data would flow through the pipeline.